sdk/communication/azure-communication-jobrouter/samples/classification_policy_crud_ops.py

```diff
@@ -0,0 +1,176 @@
+                            )
+                        ]
+                    ),
+                    ConditionalWorkerSelectorAttachment(
+                        condition = ExpressionRule(expression = "If(job.HighPriority = \"true\", true, false)"),
+                        label_selectors = [
+                            WorkerSelector(
+                                key = "Skill_O365_Lvl",
+                                label_operator = LabelOperator.GREATER_THAN_EQUAL,
+                                value = 10
+                            )
+                        ]
+                    )
+                ]
+            )
+        )
+
+        print(f"Classification Policy successfully created with id: {classification_policy.id}")
+
+        # [END create_classification_policy]
+
+    def update_classification_policy(self):
+        connection_string = self.endpoint
+        policy_id = self._cp_policy_id
+        # [START update_classification_policy]
+        from azure.communication.jobrouter import (
+            RouterAdministrationClient,
+            ClassificationPolicy,
+            ExpressionRule,
+        )
+
+        # set `connection_string` to an existing ACS endpoint
+        router_admin_client = RouterAdministrationClient.from_connection_string(conn_str = connection_string)
+        print("RouterAdministrationClient created successfully!")
+
+        updated_classification_policy: ClassificationPolicy = router_admin_client.update_classification_policy(
+            classification_policy_id = policy_id,
+            prioritization_rule = ExpressionRule(
+                expression = "If(job.HighPriority = \"true\", 50, 10)"
+            )
+        )
+
+        print(f"Classification policy successfully update with new prioritization rule")
+        # [END update_classification_policy]
+
+    def get_classification_policy(self):
+        connection_string = self.endpoint
+        policy_id = self._cp_policy_id
+        # [START get_classification_policy]
+        from azure.communication.jobrouter import RouterAdministrationClient
+
+        router_admin_client = RouterAdministrationClient.from_connection_string(conn_str = connection_string)
+
+        classification_policy = router_admin_client.get_classification_policy(classification_policy_id = policy_id)
+
+        print(f"Successfully fetched classification policy with id: {classification_policy.id}")
+        # [END get_classification_policy]
+
+    def list_classification_policies(self):
+        connection_string = self.endpoint
+        # [START list_classification_policies]
+        from azure.communication.jobrouter import RouterAdministrationClient
+
+        router_admin_client = RouterAdministrationClient.from_connection_string(conn_str = connection_string)
+
+        classification_policy_iterator = router_admin_client.list_classification_policies(results_per_page = 10)
+
+        for policy_page in classification_policy_iterator.by_page():
```

Comment:
It looks like all the samples are demonstrating by_page. If we need to - have maybe one or two samples doing this where there might be a need to process something in batches, but otherwise just iterate the results directly.

Reply:
I have added additional samples for all entities for both cases of list simply iterating over the paged obj and by batches for the sake of symmetry :D